test/e2e/fixture-builder.js

```diff
@@ -657,6 +643,11 @@ class FixtureBuilder {
           lastSelected: 1665507800000,
           name: 'Account 2',
         },
+        [ERC_4337_ACCOUNT]: {
```

Comment:
Curious why this was added

Reply:
seems to be unused now. removed here https://github.com/MetaMask/metamask-extension/pull/27847/commits/4c700fd261135035fa27438940d1207c7889ae2e